README.md

```diff
@@ -6,63 +6,85 @@ FreeTube is an open source desktop YouTube player built with privacy in mind.
 FreeTube uses a built in extractor to grab and serve data / videos.  The [Invidious API](https://github.com/iv-org/invidious) can also optionally be used.  FreeTube does not use any official APIs to obtain data.  While YouTube can still see your video requests, it can no
 longer track you using cookies or JavaScript. Your subscriptions and history are stored locally on your computer and never sent out.  Using a VPN or Tor is highly recommended
 to hide your IP while using FreeTube.
 
-Go to [FreeTube's Documentation](https://docs.freetubeapp.io/) if you'd like to know more about how to operate FreeTube and its features.
-
-## Screenshots
-<img src="https://i.imgur.com/zFgZUUV.png" width=300> <img src="https://i.imgur.com/9evYHgN.png" width=300> <img src="https://i.imgur.com/yT2UzPa.png" width=300> <img src="https://i.imgur.com/47zIEt4.png" width=300> <img src="https://i.imgur.com/hFB2fKC.png" width=300>
-
 ## Features
 * Watch videos without ads
 * Use YouTube without Google tracking you using cookies and JavaScript
 * Two extractor APIs to choose from (Built in or Invidious)
 * Subscribe to channels without an account
-* Local subscriptions, history, and saved videos
+* Connect to an externally setup proxy such as Tor
+* View and search your local subscriptions, history, and saved videos
 * Organize your subscriptions into "Profiles" to create a more focused feed
 * Export & import subscriptions
+* Youtube Trending
+* Youtube Chapters
+* Most popular videos page based on the set Invidious instance
+* SponsorBlock 
 * Open videos from your browser directly into FreeTube (with extension)
-* Mini Player
+* Watch videos using an external player
 * Full Theme support
+* Make a screenshot of a video
+* Multiple windows
+* Mini Player (Picture-in-Picture)
+* Keyboard shortcuts
+* Option to show only family friendly content
+* Show/hide functionality or elements within the app using the distraction free settings
 
-## Download Links
+### Browser Extension
+FreeTube is supported by the [Privacy Redirect](https://github.com/SimonBrazell/privacy-redirect) and [LibRedirect](https://github.com/libredirect/libredirect) extensions, which will allow you to open YouTube links into FreeTube. You must enable the option within the advanced settings of the extension for it to work.
 
-### Official Downloads
+* Download Privacy Redirect for [Firefox](https://addons.mozilla.org/en-US/firefox/addon/privacy-redirect/) or [Google Chrome](https://chrome.google.com/webstore/detail/privacy-redirect/pmcmeagblkinmogikoikkdjiligflglb).
+
+* Download LibRedirect for [Firefox](https://addons.mozilla.org/firefox/addon/libredirect/) or [Google Chrome](https://github.com/libredirect/libredirect/blob/master/chromium.md).
+
+If you have issues with the extension working with FreeTube, please create an issue in this repository instead of the extension repository.
+
+The following builds are supported:
 
+* .deb
+
+* .dmg
+
+* .exe
+
+* Flatpak
+
+* Portable (Windows only)
+
+* .rpm
```

Comment:
Why are these so separated?

Reply:
they need to be removed stop using ur thunderbolt so fast :'(

Reply:
No
![image](https://user-images.githubusercontent.com/1018543/194732791-08d87f66-0eb5-4082-9f5b-dd1b96664377.png)